model/transformer.py

```diff
@@ -0,0 +1,156 @@
+"""
+  Model Class for Transformer
+"""
+
+import math
+import torch
+import torch.nn as nn
+
+from torch.nn import TransformerDecoder, TransformerDecoderLayer, TransformerEncoder, TransformerEncoderLayer
+from yacs.config import CfgNode
+
+
+class PositionalEncoding(nn.Module):
+    """
+    Positional encoding for transformer model
+    Code from https://pytorch.org/tutorials/beginner/transformer_tutorial.html
+    """
+
+    def __init__(self,
+                 d_model: int,
+                 dropout=0.1,
+                 max_len=500):

```

Comment:
do you need a default value here? if yes, make it configurable, if not, delete it
Goes for:
* dropout
* max len